### PR TITLE
PNI - Addressing some primary nav link issues

### PR DESCRIPTION
**Pull request by mmmavis**
- Removed text colour override from PNI's active primary nav links (tablet/desktop).
- Fixed a bug where wavy underline is missing on Safari

Related PRs/issues: #9669 , #9701

## Things to QA
- Default state: black text with no underline
- Active state: black text with wavy blue underline
- Hover state: blue text with wavy blue underline
- Make sure wavy underline shows on Safari

### CODE REVIEW

File: source/sass/components/primary-nav.scss [hunk 1]
```diff
@@ -42,7 +42,6 @@
       &:hover {
         text-decoration: wavy $pni-wavy-blue 2px underline;
         text-underline-offset: 0.5rem;
-        color: $dark-blue;
```
Comment (mmmavis) [inline source/sass/components/primary-nav.scss:45]: hover state will still be in `$dark-blue` because of line 55

**Comment by mmmavis** [comment]
Hi @beccaklam , before I assign a dev for code review, could you help me double check if things I listed in the QA section of the PR description match the design? Thank you!

**Comment by beccaklam** [comment]
@mmmavis I think @nancyt1 is actually more familiar with this work so tagging her to check!

**Comment by beccaklam** [comment]
Hi @mmmavis the only error I found was in item three which I've crossed out and replaced with the correct colour:

## Things to QA
- Default state: black text with no underline
- Active state: black text with wavy blue underline
- Hover state: ~blue~ black text with wavy blue underline
- Make sure wavy underline shows on Safari

However there are further updates to the navigation that @nancyt1 designed. Not sure what the order of operations should be here since we are planning to move away from the squiggly line?

**Comment by mmmavis** [comment]
@beccaklam Thanks for the note. In that case I'll keep this PR as draft and wait for Nancy's changes before I wrap up this PR for code review.

**Comment by mmmavis** [comment]
PR Outdated. Closing

**Pull request closed without merging**